Assets/Scripts/Capabilities/Jump.cs

```diff
@@ -2,11 +2,15 @@
 using System.Collections.Generic;
 using UnityEngine;
 
+public enum JumpState {Rising, Falling, Grounded};
```

Comment:
nit (minor): the first enum state will be the default value - I think that Grounded would be the default here. Probably totally fine

Reply:
I just put them in the order that the animation follows, but that makes sense

Reply:
No change needed. In fact, highly recommend NOT changing enum values.

Reply:
For now these one are used exclusively within the Jump script, so if ever there was a time to reorder them, it would be now

Reply:
Sounds good, no reorder needed. Since enums are just basically integers under the hood, changing enum order will mean that, for example

```
// numbers just added as an example

enum BfastFood {
  Saugage = 0,
  Eggs = 1,
  Bacon = 2,
}

// changed to:

enum BfastFood {
  Pancakes = 0, // << added
  Saugage = 1,
  Eggs = 2,
  Bacon = 3,
}

// let's say we previously serialized this, and it was set to Bacon (2)
[SerializeField] BfastFood food;

// after changing the enum, the value would now be Eggs (2)
```

Hopefully that makes sense.

Reply:
That does make sense. Good to know for the future